portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,31 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomBibleVerse() {
+  const bibleVersesText =
+      ['Not only so, but we also glory in our sufferings, because we know that suffering produces perseverance; perseverance, character; and character, hope. And hope does not put us to shame, because God’s love has been poured out into our hearts through the Holy Spirit, who has been given to us.',
```

Comment:
Make sure your lines are 80 characters or fewer - https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Reply:
You also might want to consider constructing these strings outside the container to more easily wrap. Up to you - it depends whether you think it looks legible within the data structure (whether you decide to go with arrays or a Map, see below comment).

Reply:
Done!